README.md

```diff
@@ -23,9 +23,29 @@ Scripts
 -------
 
 * `npm run dev`: development mode (hot reloading and all that jazz)
+* `npm run fetch-plugins`
 * `npm start`: production mode
 * `npm run test`: run tests
 
+Production setup
+----------------
+
+Kerrokantasi-ui always builds itself on start. You can use your favorite
+process manager to either run `npm start` or server(/index.js)
+
+Questionnaires can make use of plugins. A default set of plugins can be
+installed using `npm run fetch-plugins -- { target_directory }`. By default,
+the plugins are installed in `assets/plugins` and served out in
+`assets/plugins` URL prefix by kerrokantasi-ui.
+
+If you wish, you can also specify a target directory and serve out that directory
+as `assets/plugins` in the same URL space as kerrokantasi-ui. The urls for
+plugins are partially hardcoded in kerrokantasi-ui/src/components/PluginContent.js
+and partially somewhere else. Maybe returned by the API?
```

Comment:
Nope, the UI should be the single source of truth of plugin urls, matching the plugin identifier (provided by the API) with the URL. The URLs are no longer present in the API.

Reply:
The identifier-URL mapping will indeed need to be hardcoded somewhere in the UI, you may suggest your favorite location for them. It's a bit weird to have them partially in `kerrokantasi-ui/src/components/PluginContent.js` and partially `deploy/fetch_plugins.js`, isn't it?

Reply:
Or perhaps it's not weird.

Reply:
Plugin location are now in src/shared_config.json
"Shared", as in shared between plugin cloner and the UI itself